rs/rosetta-api/tests/system_tests/system_tests.rs

```diff
@@ -18,3 +18,19 @@ fn smoke_test() {
         assert_eq!(res.network_identifiers.len(), 1);
     });
 }
+
+#[test]
+fn test_cc() {
+    println!(
+        "Acc: {:?}",
+        icp_ledger::AccountIdentifier::new(
+            ic_types::PrincipalId(
+                candid::Principal::from_text(
+                    "iowfl-yzooa-br3dt-77erl-nlm7f-kplhq-php75-hw3an-aeqn2-swh4t-3qe",
+                )
+                .unwrap(),
+            ),
+            None,
+        )
+    );
+}
```

Comment:
Is this a debugging leftover that should be removed?

Reply:
Yes thanks for catching that.